assets/src/edit-story/components/dialog/dialog.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { ThemeProvider } from 'styled-components';
+
+/**
+ * Internal dependencies
+ */
+import {
+  Dialog as StyledDialog,
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  theme,
+  lightMode,
+} from '../../../design-system';
+
+const Dialog = ({
+  open,
+  onClose,
+  closeText,
+  onConfirm,
+  confirmText,
```

Comment:
Could we default these texts or is that stupid? Would you always explicitly supply those rather than relying on defaults even if the texts are just "Cancel" and "Confirm"?

Reply:
Feel like there should always be text associated with these, but it doesn't hurt to have a default.